commands/DbWipe.ts

```diff
@@ -0,0 +1,67 @@
+import { BaseCommand, flags } from '@adonisjs/core/build/standalone'
+
+export default class DbWipe extends BaseCommand {
+  public static commandName = 'db:wipe'
+  public static description = 'Drop all tables in database'
+
+  /**
+   * Choose a custom pre-defined connection. Otherwise, we use the
+   * default connection
+   */
+  @flags.string({ description: 'Define a custom database connection', alias: 'c' })
+  public connection: string
+
+  /**
+   * Force run migrations in production
+   */
+  @flags.boolean({ description: 'Explicitly force to run migrations in production' })
```

Comment:
We may have to change the description of the flag

Reply:
Nice catch ! Done